projects/nginx/add_fuzzers.diff

```diff
@@ -0,0 +1,30 @@
+diff --git a/src/os/unix/ngx_errno.c b/src/os/unix/ngx_errno.c
+index e787b237..321c3215 100644
+--- a/src/os/unix/ngx_errno.c
++++ b/src/os/unix/ngx_errno.c
+@@ -85,3 +85,13 @@ failed:
+ 
+     return NGX_ERROR;
+ }
++
++void
```

Comment:
probably put these under `#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION`, then it may make it to the upstream repo